packages/augur-core/source/contracts/reporting/IUniverse.sol

```diff
@@ -51,13 +52,14 @@ contract IUniverse is ITyped {
     function isContainerForMarket(IMarket _shadyTarget) public view returns (bool);
     function isContainerForReportingParticipant(IReportingParticipant _reportingParticipant) public view returns (bool);
     function isContainerForShareToken(IShareToken _shadyTarget) public view returns (bool);
-    function addMarketTo() public returns (bool);
-    function removeMarketFrom() public returns (bool);
+    function migrateMarketOut(IUniverse _destinationUniverse) public returns (bool);
```

Comment:
are these methods for V3 migrating markets?

Reply:
Migrating to child universes after a fork